### PR TITLE
CTC Benchmarking code

### PR DESCRIPTION
**Pull request by vineelpratap**
Compare CTC performance with torch. Default values in the script correspond to a LibriSpeech example with stride 8 AM and batchsize=32 and 10k Word Pieces. 

Here are some initial numbers  (on Nvidia P100)
> python bindings/python/benchmarks/ctc.py -N 10 

Screen Shot 2021-12-28 at 1 37 34 PM

### CODE REVIEW

File: bindings/python/benchmarks/ctc.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import argparse
+import torch
+import torch.nn
+import time
+import gtn
+import gtn.criterion
+
+
+parser = argparse.ArgumentParser("CTC Benchmark with GTNs")
```
Comment (awni) [inline bindings/python/benchmarks/ctc.py:9]: There is a `time_utils.py` which you can use to do some of the basics in timing (warmup and printing results etc). You can also add the cuda synchronize there since it should be part of any timing.

**Review by awni — APPROVED** [review]

**Comment by vineelpratap** [comment]
Addressed all comments
